packages/snap-store-mobx/src/Search/Stores/SearchResultStore.ts

```diff
@@ -159,6 +172,13 @@ export class Product {
 		makeObservable(this.mappings.core!, coreObservables);
 	}
 
+	public getCalloutBadge(name: string): ResultBadge | undefined {
+		return this.badges.find((badge) => badge.type === 'callout' && badge.location === name);
+	}
+	public getOverlayBadges(): ResultBadge[] {
+		return this.badges.filter((badge) => badge.type === 'overlay');
+	}
+
```

Comment:
I'd prefer that these go away or somewhere more namespaced, eg: result.badges.overlay / result.badges.callout